include/node.hpp

```diff
@@ -18,6 +18,26 @@ using namespace fair::literals;
 namespace stdx = vir::stdx;
 using fair::meta::fixed_string;
 
+template<typename F>
+[[gnu::always_inline]] constexpr void
+simd_epilogue(auto width, F &&fun) {
+    static_assert(std::has_single_bit(+width));
+    auto w2 = meta::ic<width / 2>;
```

Comment:
please inline the full `std::integral_constant` expression. The short-hand notation is confusing/obfuscating what is being meant here.... also we are not hitting any column limits. :-)

Reply:
I'm not sure whether hiding the important part (`width / 2`) inside a lot of characters helps. IMHO the direct use of `std::integral_constant<std::size_t, width / 2>{}` is obfuscating. But I'm fine either way.